dev/integration_tests/release_smoke_test/lib/main.dart

```diff
@@ -11,9 +14,16 @@ void main() {
   // The test will grep logcat for any errors emitted by Flutter.
   print(text.toDiagnosticsNode());
   print(text.toStringDeep());
+  // regression test for https://github.com/flutter/flutter/issues/49601
+  final List<int> computed = await compute(_utf8encode, 'test', debugLabel: null);
+  print(computed);
```

Comment:
Unfortunately, this is the only way to test release mode code right now.  See https://github.com/flutter/flutter/issues/34505